dds_registration/migrations/0001_initial.py

```diff
@@ -25,27 +25,59 @@ class Migration(migrations.Migration):
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('code', models.TextField(default=dds_registration.models.random_code, unique=True)),
                 ('title', models.TextField(unique=True)),
-                ('description', models.TextField(blank=True)),
-                ('public', models.BooleanField(default=False)),
+                ('description', models.TextField()),
+                ('success_email', models.TextField()),
```

Comment:
@cmutel Can't see a description for this field. Can't see its usage anywhere in the app. Why is it required? Which email(s) should be here?

Reply:
This is a text field for an email which should be sent once the registration is confirmed (i.e. the person is selected and the registration fee is paid). See https://github.com/Depart-de-Sentier/dds_registration/issues/128